### ISSUE TITLE
GATConv Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

### ISSUE THREAD

**Issue by royabel** [rusty1s/pytorch_geometric#654]
## 🐛 Bug

I am trying to run GATConv layer and getting Process finished with exit code 139 (interrupted by signal 11: SIGSEGV).
It is happens also when I am running your example code at https://github.com/rusty1s/pytorch_geometric/blob/master/examples/gat.py.

The error occurs at the softmax step in the forward, and more specifically at torch_scatter.max 
line 13: func(src, index, out, arg, dim)

## To Reproduce

1. run your code example at https://github.com/rusty1s/pytorch_geometric/blob/master/examples/gat.py
2. Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

## Environment

* OS: centos 7
* Python version: 3.7
* PyTorch version: 1.1.0
* torch geometric version: 1.3.0
* torch scatter version: 1.3.1
* CUDA/cuDNN version: 8.0.61
* GCC version: 4.8.5

## Additional context

failure occurs at the softmax step -> ScatterMax.apply -> func(src, index, out, arg, dim)

**Comment by rusty1s** [comment]
This generally indicates an installation error with `torch-scatter`. You can try to reinstall the package and look for errors and should validate your installation by running the test suite of `torch-scatter`.

**Comment by royabel** [comment]
Thanks, my gcc version wasn't fit.
I upgraded the gcc, reinstall torch scatter, and now it is working
